### ISSUE TITLE
ARM64 image is missing for 11-jre

### ISSUE THREAD

**Issue by thof** [adoptium/containers#50]
I noticed the `linux/arm64/v8` image is missing for [eclipse-temurin:11-jre](https://hub.docker.com/layers/eclipse-temurin/library/eclipse-temurin/11-jre/images/sha256-87d6207d6e6c6d24acab2608e4f0152d240fe09532b11714000867b6b0d01b22?context=explore) but it is present for `8-jre`. 

Is there any reason preventing you from building ARM64 image for 11-jre? Currently it's a bit inconsistent I must say.

**Comment by smlambert** [comment]
No 11-jre has been released yet out of the project (therefore no containers have been produced yet).

11-jre should get released today (as per announcement in release channel in Slack: https://adoptium.slack.com/archives/CLCFNV2JG/p1631015141035300).  

What is the reason for the delay?  Each binary we release needs to be TCK and AQA tested, the July release took hundreds of hours of testing to complete, on shared resources by a small set of people.  We announced that we would prioritize jdks over jres, and 8-jres over 11-jres (which we discussed not releasing at all in this issue: https://github.com/adoptium/temurin-build/issues/2683).

For more details on status of the release, see https://github.com/adoptium/adoptium/issues/68 where each 👍 represents around 50hrs of testing, and requires subsequent triage and reruns in the event of unexpected failures, sometimes queued up on the single machine that is available.  The entire release then is an initial ~2500hrs of testing, followed by unknown amount of time if there are failed tests to triage, debug, fix, rerun.
